README.md

```diff
@@ -22,7 +22,18 @@ npm install react-sticky
 
 ## Overview & Basic Example
 
-`<Sticky />` elements require a function that returns a React component or DOM node as the only child. When invoked, this child function will be supplied an object with the following properties:
+The goal of `react-sticky` is make it easier for developers to build UIs that have sticky elements. Some examples include a sticky navbar, or a two-column layout where the left side sticks while the right side scrolls.
+
+First, wrap the element that you want to stay sticky in a `<Sticky />` tag. Then, wrap the `<Sticky />` tag in a `<StickyContainer />`:
+
+```js
+<StickyContainer>
+  <Sticky>
+    return <h1>{"I'm sticky"}</h1>
```

Comment:
Typo I think, `return` isn't what you want. I'm thinking it might be confusing to have 


```js
<StickyContainer>
  <Sticky>
    <h1>{"I'm sticky"}</h1>
```
though, if we're going to immediately say "but now wrap it in a function please." IMO lots of people stop reading docs at the first code sample, so first code sample should work if they copy-paste.

Reply:
That was my goal. I want this to be a first step that people can copy-paste (and kind of understand what's happening without the wrapping if people don't know about function as a child).

Maybe there's a way for me to make it clearer, maybe by saying something like "Please read all the way through" and then *Step 1* *Step 2* and so on.